<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>80</y>
     <width>201</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cbox_1">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>310</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>篮球</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cbox_2">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>310</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>排球</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cbox_3">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>320</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>足球</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="rbtn_1">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>390</y>
     <width>61</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>男</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="rbtn_2">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>390</y>
     <width>51</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>女</string>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
